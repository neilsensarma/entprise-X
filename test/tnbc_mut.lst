NP_009231.2  1756    GLN  ALA
NP_009229.2  1756    GLN  ALA
NP_009225.1  1756    GLN  ALA
NP_009228.2  1756    GLN  ALA
NP_009230.2  1756    GLN  ALA
NP_000537.3   306    ARG  ALA
NP_001119584.1 306    ARG  ALA
NP_001119587.1 306    ARG  ALA
NP_001119585.1 306    ARG  ALA
NP_001119589.1 306    ARG  ALA
NP_001119586.1 306    ARG  ALA
NP_001119588.1 306    ARG  ALA
NP_000259.1    231    GLU  ALA 
NP_861971.1    231    GLU  ALA
NP_057502.2    231    GLU  ALA
NP_861966.1    231    GLU  ALA
NP_861967.1    231    GLU  ALA
NP_861968.1   231    GLU  ALA
NP_861970.1   231    GLU  ALA
NP_861546.1   231    GLU  ALA
NP_861969.1   231    GLU  ALA
NP_004351.1   638   TRP  ALA
