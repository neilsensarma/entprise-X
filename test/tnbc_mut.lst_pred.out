NP_009231.2 1756 0.901484
NP_009225.1 1756 1.00169
NP_009228.2 1756 0.83442
NP_000537.3 306 0.925695
NP_001119584.1 306 0.925695
NP_001119585.1 306 0.836607
NP_001119586.1 306 0.823876
NP_000259.1 231 0.980092
NP_057502.2 231 0.957977
NP_861966.1 231 0.939321
NP_861967.1 231 0.933707
NP_861968.1 231 0.841749
NP_861970.1 231 0.957977
NP_861546.1 231 0.957977
NP_861969.1 231 0.841749
NP_004351.1 638 0.93915
